__tests__/app.test.js

```diff
@@ -91,3 +91,64 @@ describe('GET /api/articles', () => {
     });
 })
 
+describe('POST /api/articles/:article_id/comments', () => {
+    test('resolves with status 201 and correct comment object.', () => {
+        const newComment = {
+            username: 'lurker',
+            body: 'not a bad read'
+        }
+        return request(app).post('/api/articles/1/comments').send(newComment).expect(201).then((res) => {
+            const comment = res.body.comment;
+            expect(comment).toEqual(newComment.body);
```

Comment:
for the sake of whatever might be using the comment data when the posted data comes back I would include the comment_id in the response. Imagine you're building a frontend and rely on this response to make a article page. You would want to know how to identify this comment in a list

Reply:
NOTE for myself: 

I want you to return the ENTIRE comment that was posted, this includes votes, created_at, the lot.